custom_components/tuya_local/devices/pir_sensor_alarm.yaml

```diff
@@ -0,0 +1,69 @@
+name: PIR sensor alarm
+products:
+  - id: y5tdmwmb2ieknmkb
+entities:
+  - entity: binary_sensor
+    translation_key: motion
+    class: motion
+    dps:
+      - id: 1
+        type: string
+        name: sensor
+        mapping:
+          - dps_val: pir
+            value: true
+          - dps_val: "none"
+            value: false
+  - entity: switch
+    name: Armed
+    icon: "mdi:shield-home"
+    dps:
+      - id: 104
+        type: boolean
+        name: switch
+  - entity: binary_sensor
+    translation_key: charge_energy
```

Comment:
There is no translation key by this name, but maybe `class: battery_charging` is what you really want here.